### DISCUSSION TITLE
change route group convention from parentheses to something more cli friendly

### DISCUSSION THREAD

**Discussion by HansKramer** [vercel/next.js#71554] [App Router]
Parentheses are not very convenient to the command-line.
And personally, I dislike IDE's with a passion.

Is there a configuration option to change this "convention" to some
thing more cli friendly like underscores or other characters that don't have to be escaped by the shell?

Kind regards

Hans.

**Comment by icyJoseph** [comment]
AFAIK is not possible to change this... Searching files with `\(` is indeed not that nice. Also, `\[` is not very nice either, but most terminal emulators seem to autocomplete it correctly.
